fizz-cli/fizz_cli/main.py

```diff
@@ -1,30 +1,24 @@
 import time
-
 import typer
 
-# TODO: These are the bullet imports
-from bullet import Bullet
-from bullet import Input
-from bullet import YesNo
-from bullet import colors
-
-# TODO: -------
 from click import clear
 from rich import print
 
-from .utils import bold_blue
-from .utils import check_fission_directory
-from .utils import delete_file_if_exists
-from .utils import ensure_leading_slash
-from .utils import enumerate_functions
-from .utils import get_fn_route_path
-from .utils import read_yaml_file
-from .utils import rename_fn_in_specs
-from .utils import rename_folder
-from .utils import replace_route
-from .utils import save_yaml_file
-from .utils import styled_bullet
-from .utils import update_shell_scripts
+from .utils import (
```

Comment:
OPTIONAL: add black connect to your ide, it will pick settings from pyproject.toml for formatting. I prefer one import per line and it's that way in all the projects so this will cause issues and conflicts if we don't stick to a preset way of doing imports